metabase/metabase
# Remove animation from ViewSidebar under reduce motion

One more step to implement issue #17726

## How to Test

1. Ask a Question
2. Simple question
3. Sample dataset
4. Orders
5. Click on
  a. Visualization
  b. Settings
  c. Filter
  d. Summarize
  
to open and close their sidebars.
 
If you have `prefers-reduced-motion` [enabled](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion#user_preferences), opening and closing the sidebars should not be animated.
